### ISSUE TITLE
Suppress shelfmarks for archives and manuscripts

### ISSUE THREAD

**Issue by jtweed** [wellcomecollection/platform#5167]
As part of #5147 we should suppress all shelfmarks on archives and manuscripts.

These are problematic, as they contain a duplicate of the reference number plus a box number, which is both confusing and something we don't to make publicly available.

We should do this before making item data available on works pages.

**Comment by alexwlchan** [comment]
I'm assuming "on archives and manuscripts" here means Sierra material type code `h` ("archives and manuscripts").

**Comment by jtweed** [comment]
Yes I think we can assume that. I believe it's technically the shelfmarks that are created during the Calm->Sierra harvest that have this format, but it's still a group we'll want to suppress across the board.

**Comment by alexwlchan** [comment]
> Yes I think we can assume that

Woohoo, cos that's how I implemented it. 😅
